solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -627,11 +659,7 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
         setReserveBalances(sourceId, targetId, actualSourceBalance, targetBalance - amount);
 
         // transfer funds to the beneficiary in the to reserve token
-        if (_targetToken == NATIVE_TOKEN_ADDRESS) {
-            _beneficiary.transfer(amount);
-        } else {
-            _targetToken.safeTransfer(_beneficiary, amount);
-        }
+        safeTransfer(_targetToken, _beneficiary, amount);
```

Comment:
might be worth adding a private function that checks for `NATIVE_TOKEN_ADDRESS` and handles transferring it interrnally

Reply:
That's exactly what function `safeTransfer` is and does, so I don't understand the comment.